package.json

```diff
@@ -8,18 +8,17 @@
   },
   "scripts": {
     "clean": "rm -rf build/ .parcel-cache",
-    "start": "REACT_APP_META_CSP= parcel --port 3000 --target web --dist-dir build",
     "parcel:prod": "parcel build --target web --dist-dir build",
     "clean:ext": "rm -rf build-ext/ .parcel-cache",
-    "start:ext": "HMR_PORT=2222 EXTENSION_CSP=`yarn --silent print-extension-csp` parcel --host localhost --target ext --dist-dir build-ext",
+    "start": "REACT_APP_META_CSP= HMR_PORT=2222 EXTENSION_CSP=`yarn --silent print-extension-csp` parcel --host localhost --port 3000 --dist-dir build-dev",
     "build:ext": "yarn clean:ext && EXTENSION_CSP=`yarn --silent print-extension-csp` parcel build --target ext --dist-dir build-ext && node ./internals/scripts/validate-ext-manifest.js",
```

Comment:
Hm. Extension is broken if built with `yarn build:ext`, but it works from `yarn start`

Reply:
In background page: `Uncaught Error: Cannot find module 'fj44w'`
background.8e475e86.js contains: `"webext-redux": "fj44w",`
popup.9c7b8803.js contains:  `"webext-redux": "fj44w",`
popup.c6ac691b.js contains: `fj44w: [ function(q, H, ee) {`

Reply:
Upstream issue https://github.com/parcel-bundler/parcel/issues/8071#issuecomment-1148069991

Reply:
Deferring to https://github.com/oasisprotocol/oasis-wallet-web/issues/853